src/main/scala/org/ergoplatform/nodeView/wallet/DefaultBoxSelector.scala

```diff
@@ -57,7 +58,9 @@ object DefaultBoxSelector extends BoxSelector {
     }
 
     def balanceMet = currentBalance >= targetBalance
-    def assetsMet = targetAssets.forall { case (id, targetAmt) => currentAssets.getOrElse(id, 0L) >= targetAmt }
+    def assetsMet = targetAssets
+      .filterNot { case (id, _) => res.headOption.exists(bx => Algos.encode(bx.id) == id) }
```

Comment:
What is this for?

Reply:
@catena2w, Box selector considered outputs with issuing assets when calculating targeted asset balance, this led to selection failure. This is for excluding new assets.